src/Program.cs

```diff
@@ -98,7 +151,12 @@ public static async Task<int> Run(string? folder, string? workspace, string? ver
                 string workspaceDirectory;
                 string workspacePath;
                 WorkspaceType workspaceType;
+                if (!string.IsNullOrEmpty(workspace) && string.IsNullOrEmpty(project))
+                {
+                    logger.LogWarning(Resources.Workspace_option_is_deprecated_Use_the_project_argument_instead);
+                }
 
+                workspace ??= project;
                 if (!string.IsNullOrEmpty(folder) && !string.IsNullOrEmpty(workspace))
```

Comment:
This check is now done in the validator (`ValidateWorkspaceAndFolder`). Should I remove it here?

Reply:
Please do. =)